Samples/src/VisualDebugger/Hubs.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.SignalR;
+using System.Threading.Tasks;
+
+namespace vis_sim
+{
+    internal class VisualHub : Hub
```

Comment:
Let's add some ///summary with what this class does and how it works, for those not familiar with SignalR.